src/components/search/SearchField.js

```diff
@@ -2,18 +2,21 @@ import React, { useState, useEffect } from 'react';
 import { Link } from 'react-router-dom';
 import { Button, Modal, Card } from 'antd';
 
+import '../search-bar/Modal.css';
```

Comment:
@PaulMEdwards remember to replace this CSS reference with the LESS version after this PR is merged to main.